### PR TITLE
[CFD-32] Add Activity Model

### PR DESCRIPTION
**Pull request by helenaglow**
### Description
Created an Activity Model in `schema.prisma` with:
- [X] `name`: `String` (as primary key)
- [X] `day`: `DateTime @db.Date` (_only shows date portion of timestamp_)
- [X] `startTime` : `DateTime @db.Time()` (_only shows time_)
- [X] `endTime` : `DateTime @db.Time()` (_only shows time_)
- [X] `leader` : `String`
- [X] `location` : `String`

(I have not migrated it yet as it says to only do it once per PR, so I will wait for approval first)

Completes CFD-32
Contributes to CFD-28

### Type of change

- [X] New feature (non-breaking change which adds functionality)

### Testing

The video shows changing the time on the `day` time stamp and it reverting as it is not needed. Same goes for `startTime` and `endTime`. 

https://github.com/uoftblueprint/centre-for-dreams/assets/113125436/5b862043-103d-4b21-9545-ab639f424e5a

### CODE REVIEW

**Comment by helenaglow** [comment]
I added `id` so that it is easier to change the activity name.

**Review by BakerWJ — APPROVED** [review]

File: packages/db/prisma/schema.prisma [hunk 1]
```diff
@@ -31,3 +31,13 @@ model Comment {
   text      String
   createdAt DateTime @default(now())
 }
+
+model Activity {
+  id        Int      @id
```
Comment (BakerWJ) [inline packages/db/prisma/schema.prisma:36]: add autoincrement to id

**Review by BakerWJ — CHANGES_REQUESTED** [review]
rip sorry just realized that `id` didn't have a default value. will approve after small fix

**Comment by helenaglow** [comment]
Ahh I forgot that oops, thank you for catching 😅

**Comment by BakerWJ** [comment]
Can you make a migration? Will approve once migration is created

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**